docs/manual/manual_telemetry.rst

```diff
@@ -0,0 +1,235 @@
+	+-------------+------------------------------------+---------------------------------------------------------------------------------------------------------------+
+	| Vertiq 2306 | .. centered:: |:warning:|          | Vertiq 2306 modules do not support DroneCAN, so the DroneCAN telemetry described below does not apply to them.|
+	+-------------+------------------------------------+---------------------------------------------------------------------------------------------------------------+
+
+Servo Modules
+**************
+
+.. table:: Servo Module Support for Configurable Timeout
+
+	+-------------+------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+	| Module      | Telemetry Support                  | Notes                                                                                                                                                                              |
+	+-------------+------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+	| Vertiq 8108 | .. centered:: |:x:|                | Servo modules do not support specialized telemetry messages such as those detailed here, but it is possible to access the same information using IQUART messages on servo modules. |
+	+-------------+------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+	| Vertiq 4006 | .. centered:: |:x:|                | Servo modules do not support specialized telemetry messages such as those detailed here, but it is possible to access the same information using IQUART messages on servo modules. |
+	+-------------+------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+	| Vertiq 2306 | .. centered:: |:x:|                | Servo modules do not support specialized telemetry messages such as those detailed here, but it is possible to access the same information using IQUART messages on servo modules. |
+	+-------------+------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Supported Telemetry
+=====================
+The structure of the supported telemetry messages and how to trigger them varies between different protocols. The sections below cover these details for different protocols.

```

Comment:
The structure and triggering of supported telemetry messages varies between different protocols.